lute/static/js/lute.js

```diff
@@ -369,7 +369,6 @@ let move_cursor = function(shiftby) {
   save_curr_data_order(target);
   apply_status_class(target);
   $(window).scrollTo(target, { axis: 'y', offset: -150 });
-  show_term_edit_form(target, { autofocus: false });
```

Comment:
Hi @Jayanth-Parthsarathy - 99% sure this line is still needed, just the `{ autofocus: false }` part should be removed ... right?  The form should be shown if the user clicks a term.